docs/content_management/data_migration/exporting_data.md

```diff
@@ -136,9 +136,8 @@ Note that you should test your migrations. See [Importing data](importing_data.m
 
 !!! caution
 
-    `--siteaccess` option usage can be relevant on multi-languages repository.
-    You must export with the SiteAccess supporting all the languages, or migration skips translations in non-supported languages.
-    It is recommended to use the SiteAccess of the Back Office of the targeted repository.
+    [`--siteaccess` option](#siteaccess) usage can be relevant when multiple languages or multiple repositories are used.
+    Translations could be lost if a SiteAccess not supporting all your languages is used.
```

Comment:
```suggestion
    To prevent translation loss, a SiteAccess having all the languages must be use (often a Back Office one).
```

Reply:
```suggestion
    To prevent translation loss, it is recommended that you use the SiteAccess that has all the languages used in your implementation, most likely the Back Office one.
```